### PR TITLE
[fix bug 1178217] Updates to Hello FTU for Firefox 40

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
Still waiting one some GA testing on demo2, but this should now be good for code review.

File: media/js/firefox/hello/hello-ftu-minimal.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        // Only open the info panel if tab is visible
+        if (HelloFTU.documentHidden || document.hidden) {
+            return;
+        }
+        // Make sure loop icon is available before opening Hello panel (bug 1111828)
+        Mozilla.UITour.getConfiguration('availableTargets', function (config) {
+            if (config.targets && $.inArray('loop', config.targets) !== -1) {
+                Mozilla.UITour.showMenu('loop', function() {
+                    HelloFTU.helloPanelVisible = true;
+                    $(window).one('resize.hello', HelloFTU.handleResize);
```
Comment (jpetto) [inline media/js/firefox/hello/hello-ftu-minimal.js]: Do we want/need a `$document.one('click')` handler here to allow a user to dismiss the menu by clicking anywhere on the page? Clicking the Hello icon manually allows the user to dismiss by clicking anywhere on the page - asking more in terms of consistency than necessity.

**Comment by jpetto** [comment]
Quite the clean PR sir. :tophat:

Pending the one question above, r+!

File: media/js/firefox/hello/hello-ftu-minimal.js (hunk 1 above)
Comment (alexgibson) [inline media/js/firefox/hello/hello-ftu-minimal.js]: A fine suggestion :+1:

**Comment by alexgibson** [comment]
@jpetto - think this should be good. I left it as a separate commit for easier review - will squash before we merge, thanks!

**Comment by jpetto** [comment]
Changes and test updates look good. r+! :school_satchel:

**Pull request merged**